shared/profile/non-user-profile.render.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+import React from 'react'
+import {Avatar, BackButton, Box, Button, Icon, Text} from '../common-adapters'
+import {capitalize} from 'lodash'
+import {globalColors, globalStyles, globalMargins} from '../styles'
+import {platformToLogo24} from '../constants/search'
+import {AVATAR_SIZE, HEADER_SIZE} from '../profile/index.desktop'
+
+import type Props from './non-user-profile.render'
+
+const NonUserRender = (props: Props) => (
+  <Box style={styleContainer}>
+    <Box style={{...styleHeader, backgroundColor: globalColors.blue}} />
+    <Box style={{...globalStyles.flexBoxColumn}}>
```

Comment:
Ah, I skipped headerhoc because the actual-user profile doesn't use it, and I want the rendering to be identical to avoid noticeable differences between actual-user and non-user.  Agree that both of them should switch over to it though.